mne/io/brainvision/brainvision.py

```diff
@@ -51,6 +51,11 @@ class RawBrainVision(BaseRaw):
     %(preload)s
     %(verbose)s
 
+    Attributes
+    ----------
+    impedances : dict
+        A dictionary of all electrodes and their impedances.
```

Comment:
FYI we don't document the individual classes so this will only show up if you look at the doc of this class `__init__` in a terminal or IDE (which is probably slim to never). But I guess we can keep it